app-frontend/packs/components/editPage/EditForm.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <form class="ba-form-section" @submit.prevent="formSubmit">
+    <div>
+      <h3 class="ba-form-section__title">Creation points</h3>
+      <div class="ba-form-section__line "></div>
+    </div>
+    <div class="ba-form-block">
+      <label>Name banner</label>
+      <input class="ba-form-block__name-banner"
+             type="text" placeholder="Name banner"
+             :value="inputText"
+             @input="updateInput"
+      >
```

Comment:
подправь немного тут и ниже мультилайны

```suggestion
      <input 
          class="ba-form-block__name-banner"
          type="text" 
          placeholder="Name banner"
          :value="inputText"
          @input="updateInput"
      >
```